### PR TITLE
Aug trains - Calendar integration

### PR DESCRIPTION
**Pull request by cristobal-buenrostro**
Is required to add the new "**studentsOnly**" value into the **educationAddToCalendarOptions** enum.
The tasks below are covered on this PR:

- update educationAddToCalendarOptions enum values definition.
- educationassignmentdefaults.md updated the properties section with new value and specifying the default value.
- educationassignment.md updated the properties section with new value and specifying the default value.
- update changelog file to mention the value addition.

### CODE REVIEW

**Comment by jasonjoh** [comment]
Docs Build status updates of commit _[18f0f8a](https://github.com/microsoftgraph/microsoft-graph-docs/commits/18f0f8ae7a236061c68b70046f0b82011290c664)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[api-reference/beta/resources/educationassignment.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/v-cristobalb/calendar/api-reference/beta/resources/educationassignment.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/resources/educationassignment?view=graph-rest-beta&branch=pr-en-us-14132) (graph-rest-beta) |
[api-reference/beta/resources/educationassignmentdefaults.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/v-cristobalb/calendar/api-reference/beta/resources/educationassignmentdefaults.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/resources/educationassignmentdefaults?view=graph-rest-beta&branch=pr-en-us-14132) (graph-rest-beta) |
[api-reference/beta/resources/enums.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/v-cristobalb/calendar/api-reference/beta/resources/enums.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/resources/enums?view=graph-rest-beta&branch=pr-en-us-14132) (graph-rest-beta) |
[changelog/Microsoft.EducationAssignment.json](https://github.com/microsoftgraph/microsoft-graph-docs/blob/v-cristobalb/calendar/changelog/Microsoft.EducationAssignment.json) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuilduserstoragepublic.blob.core.windows.net/report/2021%5C9%5C2%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C202109022307306938-14132%5Cworkflow_report.html?sv=2018-03-28&sr=b&sig=GrgUWmDwTJuf067RAQiWdLyQ12%2BacVjd7qUHQqGhdIk%3D&st=2021-09-02T23%3A04%3A10Z&se=2021-10-03T23%3A09%3A10Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/77e68b91-cf14-3558-43ca-0d6d735c492d?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=master">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

File: api-reference/beta/resources/enums.md [hunk 1]
```diff
@@ -683,6 +683,7 @@ Namespace: microsoft.graph
 |none|
 |studentsAndPublisher|
 |studentsAndTeamOwners|
+|studentsOnly|
 |unknownFutureValue|
```
Comment (sharad-sharma-msft) [inline api-reference/beta/resources/enums.md:687]: unknownFutureValue comes before studentsOnly

Comment (cristobal-buenrostro) [reply api-reference/beta/resources/enums.md:686]: done

**Review by isvargasmsft — APPROVED** [review]
Thank you very much Cristobal!

**Pull request merged**